app/src/components/InformationTextLine.tsx

```diff
@@ -0,0 +1,89 @@
+import { Typography, Box } from '@mui/material';
+import React from 'react';
+
+interface IInformationTextLineProps {
+  children: string;
+  label?: string;
+  fontSize?: number;
+  labelBold?: boolean;
+  textBold?: boolean;
+}
+
+/**
+ * This is a component that shows a label before some text, this is usaed when you want to display a short line of text that has a sertain meaning. The meaning of the line is specified by the label
+ * @param children This is the text that will be shown after the label and should be a string
+ * @param label? this is the label that will be shwon before the text
+ * @param fontSize? this will set the
+ * @param labelBold? if true the label will be shown as bold
+ * @param textBold? if true the text will be shown as bold
+ * @returns InformationTextLine component
+ */
+export default function InformationTextLine({
```

Comment:
You should set the default props here instead:

https://www.amitmerchant.com/specifying-default-value-for-props-in-reactjs/